web/src/engine/osk/src/input/gestures/browser/subkeyPopup.ts

```diff
@@ -299,15 +303,22 @@ export default class SubkeyPopup implements GestureHandler {
    * @param   {Object}  key   HTML key element
    * @return  {Object}        callout object
    */
-  addCallout(key: KeyElement, delta: number, _Box: HTMLElement): HTMLDivElement {
+  addCallout(key: KeyElement, delta: number, host: HTMLElement, _Box: HTMLElement, isTablet: boolean): HTMLDivElement {
     delta = delta || 0;
 
-    let calloutHeight = key.offsetHeight - delta + 6;
+    // Uses content-box styling, so ignores border aspects for reported positions.
+    /**
+     * "Computed Menu Style"
+     */
+    const cms = getComputedStyle(this.element);
+
+    const STUB_HEIGHT = 6 + SUBKEY_MENU_VERT_OFFSET;
```

Comment:
* Value `6` here feels like it should be a start-of-file constant.
* And STUB_HEIGHT, given it's a constant, should also be start-of-file.